data/sohu

```diff
@@ -0,0 +1,3 @@
+sohu.com
```

Comment:
maybe these should be added into `sohu-ads`:
- `track.sohu.com`
- `pv.sohu.com`
- `adv-sv-show.focus.cn`